apps/sasquatch/src/projectDependency/java/com/microsoft/appcenter/sasquatch/activities/CustomItemAdapter.java

```diff
@@ -0,0 +1,63 @@
+package com.microsoft.appcenter.sasquatch.activities;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.BaseAdapter;
+import android.widget.ImageButton;
+import android.widget.ListAdapter;
+import android.widget.TextView;
+
+import com.microsoft.appcenter.sasquatch.R;
+
+import java.util.ArrayList;
+
+public class CustomItemAdapter extends BaseAdapter implements ListAdapter {
+    private ArrayList<String> list = new ArrayList<String>();
+    private Context context;
+
+    public CustomItemAdapter(ArrayList<String> list, Context context){
+        this.list = list;
+        this.context = context;
+    }
+
+    @Override
+    public int getCount() {
+        return list.size();
+    }
+
+    @Override
+    public Object getItem(int position) {
+        return list.get(position);
+    }
+
+    @Override
+    public long getItemId(int position) {
+        return 0;
+    }
+
+    @Override
+    public View getView(final int position, View convertView, ViewGroup parent) {
+        View view = convertView;
+        if(view == null){
+            LayoutInflater inflater = (LayoutInflater) context.getSystemService(Context.LAYOUT_INFLATER_SERVICE);
+            view = inflater.inflate(R.layout.item_view_property,null);
+        }
+
```

Comment:
Add a comment when starting a new block of code (or remove blank lines)

Reply:
Fix in new commit.